### DISCUSSION TITLE
Show private repo updates

### DISCUSSION THREAD

**Discussion by NewBee2020** [InfiniteSynthesis/InfiniteSynthesis#2] [Q&A] [answered]
Is it possible to add the private repo update at "Recent Updates" section, just like that in the contribution panel? @InfiniteSynthesis

**Comment by InfiniteSynthesis** [comment, marked as answer]
Yes you can do that by querying the github api with credentials to get private repos.

However, I don't think it's a good idea to show the updates on private repos in that others would not be able to see their names nor click the link to access them.
